articles/machine-learning/how-to-machine-learning-interpretability-automl.md

```diff
@@ -169,6 +193,48 @@ with open("myenv.yml","r") as f:
 
 ```
 
+### Write the Scoring Script
```

Comment:
Nit: capitalization should be consistent with the rest of the docs

Reply:
Done